server/src/db/migrations/20220222200523_rooms_table.ts

```diff
@@ -0,0 +1,69 @@
+import { Knex } from "knex";
+
+
+export async function up(knex: Knex): Promise<void> {
+    knex.schema.hasTable('Rooms').then(function(exists) {
+        if (!exists) {
+            return knex.schema.createTable('Rooms', function(t) {
+                t.increments('id').primary();
+                t.string("name", 100);
+            });
+        }
+    });
+
+    knex.schema.hasTable('EquipmentForRooms').then(function(exists) {
+        if (!exists) {
+            return knex.schema.createTable('EquipmentForRooms', function(t) {
+                t.increments('id').primary();
+                t.integer("roomID").references('id').inTable('Rooms');
+                t.integer('equipmentID').references('id').inTable("Equipment");
+            });
+        }
+    });
+
+    knex.schema.hasTable('LabbiesForRooms').then(function(exists) {
```

Comment:
do you still need this table after adding the monitoring room field to the user table?